enhancements/observability/multicluster-observability-addon.md

```diff
@@ -0,0 +1,767 @@
+  - "@iblancasa"
+reviewers:
+  - "@alanconway"
+  - "@jcantrill"
+  - "@xperimental"
+  - "@jparker-rh"
+  - "@berenss"
+  - "@bjoydeep"
+approvers:
+  - "@alanconway"
+  - "@jcantrill"
+  - "@xperimental"
+api-approvers:
+  - "@alanconway"
+  - "@jcantrill"
+  - "@xperimental"
+creation-date: 2023-11-28
+last-updated: 2023-11-28
+tracking-link:
+  - https://issues.redhat.com/browse/OBSDA-356
+  - https://issues.redhat.com/browse/OBSDA-393
+  - https://issues.redhat.com/browse/LOG-4539
+  - https://issues.redhat.com/browse/OBSDA-489
+see-also:
+  - None
+replaces:
+  - None
+superseded-by:
+  - None
+---
+
+# Multi-Cluster Logs and Traces Forwarding
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Multi-Cluster Observability has been an integrated concept in Red Hat Advanced Cluster Management (RHACM) since its inception but only incorporates one of the core signals, namely metrics, to manage fleets of OpenShift Container Platform (OCP) based clusters (See [RHACM Multi-Cluster-Observability-Operator (MCO)](rhacm-multi-cluster-observability)). The underlying architecture of RHACM observability consists of a set of observability components to collect a dedicated set of OCP metrics, visualizing them and alerting on fleet-relevant events.
+
+This enhancement proposal seeks to bring a unified approach to collect and forward logs and traces from a fleet of OCP clusters based on the RHACM addon facility (See Open Cluster Management (OCM) [addon framework](ocm-addon-framework)) by enabling these signals events to land on third-party managed storage solutions (e.g. AWS Cloudwatch, Google Cloud Logging, Splunk). The proposed addon is an optional RHACM component. It is installed by the MCO Operator. It provides a unified installation approach of required dependencies (e.g. operator subscriptions) and resources (custom resources, certificates, CA Bundles, configuration) on the managed clusters to collect and forward logs and traces. The addon's name is MultiCluster-Observability-Addon (short MCOA).
+
+## Motivation
+
+The main goal here is to simplify and unify the installation of logs and traces collection and forwarding on a RHACM managed fleet of OCP clusters. The core function of the addon is to install required operators (i.e. [Red Hat OpenShift Logging](ocp-cluster-logging-operator) and [Red Hat OpenShift distributed tracing platform](opentelemetry-operator)), configure required custom
+resources (i.e. `ClusterLogForwarder`, `OpenTelemetryCollector`) and reconcile per-cluster-related authentication resources (i.e. `Secrets`, `ConfigMaps`). This enables fleet-wide control of logs and traces collection and forwarding.
+
+### User Stories
```

Comment:
Thanks @iblancasa, this set of user stories do seem to me a bit outdated from the latest discussions we had, also taking into account that this initiative started from logging, so it's completely understandable that, from that point of view, the same _could_ be expected from tracing. Apart from that, things have evolved a lot since this initiative started, and our support in the Red Hat build of OpenTelemetry has expanded. Hence the user stories could benefit of redefinition without really impacting the mechanisms to make it happen. I'm also adding the therd part of the regular user story definition: "SO THAT" to help understanding the bigger picture and what we are trying to achieve here.

My 2 cents on what I would expect from a product perspective:
* AS A fleet administrator, I WANT TO install, configure and manage an OpenTelemetry collector on any set of RHACM managed OCP clusters SO THAT I can collect, process and export the signals already available in the Red Hat build of OpenTelemetry in a homogeneous way.

Something very similar can easily apply to the authentication of the OpenTelemetry collector.

I hope this helps! Thanks!

Reply:
I feel the current user stories are abstract enough to encapsulate the user story mentioned here, hence I'm resolving this conversation, if you disagree and want me to explicitly add this user story feel free to unresolve this conversation.